apps/viewer/src/app.tsx

```diff
@@ -177,28 +182,34 @@ const App = () => {
                 }}
               />
             </Box>
-            <VStack height="100%" maxWidth="20vw">
-              {Array.from(remoteTrackSources).map(([sourceId, { mediaStream }]) => (
-                <Box
-                  cursor="pointer"
-                  height={`calc(100% / ${MAX_SOURCES})`}
-                  key={sourceId}
-                  onClick={() => setMainSourceId(sourceId)}
-                  test-id="millicastVideo"
-                  width="100%"
-                >
-                  <ViewerVideoView
-                    isStreaming={isStreaming}
-                    videoProps={{
-                      displayVideo: true,
-                      label: sourceId,
-                      mediaStream,
-                      muted: true,
-                    }}
-                  />
-                </Box>
-              ))}
-            </VStack>
+            {mainSourceId && remoteTrackSources.size > 1 ? (
```

Comment:
When there is only one track, we should project it to main stream directly and ignore it in the `active` event to prevent we project it twice.